packages/wallet/test/hardware/ledger/LedgerKeyAgent.test.ts

```diff
@@ -37,6 +37,14 @@ const cleanupEstablishedConnections = async () => {
   LedgerKeyAgent.deviceConnections = [];
 };
 
+const getStakeCredential = (rewardAccount: Cardano.RewardAccount) => {
+  const stakeKeyHash = Cardano.RewardAccount.toHash(rewardAccount);
+  return {
+    hash: Crypto.Hash28ByteBase16.fromEd25519KeyHashHex(stakeKeyHash),
+    type: Cardano.CredentialType.KeyHash
+  };
+};
+
```

Comment:
Maybe this should not be a part of the test. If you don't need it in mapper logic and it is necessary just for tests a better place would be `./testData`

Reply:
but testData is in a separate package?